src/Container/ServerRequestFactoryFactory.php

```diff
@@ -46,6 +45,6 @@ public function __invoke(ContainerInterface $container): callable
             ? $container->get(FilterServerRequestInterface::class)
             : null;
 
-        return static fn(): ServerRequest => ServerRequestFactory::fromGlobals(null, null, null, null, null, $filter);
+        return static fn(): ServerRequestInterface => ServerRequestFactory::fromGlobals(requestFilter: $filter);
```

Comment:
Do we even support named parameters?

Reply:
- tests are passing
- PHP 8.0 introduced named parameters and thus, from a technical perspective, yes
- it actually prevents me from struggling with the 120 characters per line limit
- there is no codestyle definition which yells at me

So overall, I'd say yes.